Bender.lock

```diff
@@ -34,6 +34,13 @@ packages:
     - common_verification
     - register_interface
     - tech_cells_generic
+  axi_mem_if:
+    revision: 3567273b4f67bd57a81a2a503394a49641bae53a
+    version: 0.2.1
+    source:
+      Git: git@github.com:pulp-platform/axi_mem_if.git
```

Comment:
Please avoid using SSH links, use HTTPS.

Reply:
axi_mem is redundant in idma version ethernet. already got removed in most recent cleanup